src/utilities/Utilities.js

```diff
@@ -11,8 +11,6 @@ const concatStream = require("concat-stream")
 const Readable = require("stream").Readable
 const SerializerJsonld = require("@rdfjs/serializer-jsonld-ext")
 
-export const defaultLanguageId = "eng"
-
 export const isResourceWithValueTemplateRef = (property) =>
```

Comment:
as requested by Justin L, moved this value from `Utilities` to `Config` -- and then adjustments to it's use